docs/spark/spark-ddl.md

```diff
@@ -171,6 +171,14 @@ Iceberg uses table properties to control table behavior. For a list of available
 ALTER TABLE prod.db.sample UNSET TBLPROPERTIES ('read.split.target-size')
 ```
 
+Set table comment using SET PROPERTIES:
```

Comment:
should be 'SET TBLPROPERTIES' right?  

Also maybe :
```To set table comment using 'SET TBLPROPERTIES'```
(as the earlier sentences were not using imperative form, doesn't flow well if we switch here)

Reply:
It would be better, thanks @szehon-ho